py3status/modules/rainbow.py

```diff
@@ -191,8 +206,12 @@ def rainbow(self):
                 obj['color'] = color
             output.append(obj)
 
-        response = {'cached_until': self._cycle_time, 'composite': output}
-        return response
+        rainbow = self.py3.build_composite(self.format, composites={'rainbow': output})
```

Comment:
`build_composite()` is deprecated as we can just use `safe_format()`

eg.
`rainbow = self.py3.build_composite(self.format, {'rainbow': output})`